modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/ModelBuilderContext/objectFolderReducer.ts

```diff
@@ -192,12 +194,12 @@ export function ObjectFolderReducer(state: TState, action: TAction): TState {
 			> | null = null;
 
 			const newObjectDefinitionNodes = objectDefinitionNodes.map(
-				(objectDefinitionNode) => {
+				(node) => {
```

Comment:
@danvb112 You shouldn't change this one because of the pattern I mentioned before.